examples/macos/mac_os_calculator.py

```diff
@@ -0,0 +1,59 @@
+# GUI Application automation and testing library
```

Comment:
Please rename `macos/mac_os_calculator.py` to `macos/calculator.py`. Folder name already points to macOS.